vllm/model_executor/layers/fused_moe/flashinfer_trtllm_moe.py

```diff
@@ -23,26 +23,24 @@ def flashinfer_fused_moe_blockscale_fp8(
     w2_weight_scale_inv: torch.Tensor,
     global_num_experts: int,
     top_k: int,
-    num_expert_group: int,
-    topk_group: int,
+    num_expert_group: int | None,
+    topk_group: int | None,
     intermediate_size: int,
     expert_offset: int,
     local_num_experts: int,
     block_shape: list[int],
-    routed_scaling: float = 1.0,
+    routed_scaling: float | None = 1.0,
+    routing_method_type: int = 2,
```

Comment:
Can you use the `RoutingMethodType` enum here?  And the other places using raw ints.